terraform/dev/main.tf

```diff
@@ -38,8 +39,8 @@ module "vpc" {
   master_ipv4_cidr_block = var.gke["master_ipv4_cidr_block"]
 }
 
-module "gke_service_account" {
-  source = "git::git@github.com:cloudkite-io/terraform-modules.git//modules/gke-service-account?ref=v0.0.1"
-  name = "${module.gke.name}-sa"
+module "velero_service_account" {
+  source = "git::git@github.com:cloudkite-io/terraform-modules.git//modules/velero-service-account?ref=v0.0.1"
```

Comment:
This doesn't reference what your latest PR calls the velero module.

Reply:
Closing this PR. Last PR has all latest changes.